examples/simple-nightwatch/SimpleNightwatch.elm

```diff
@@ -73,6 +73,9 @@ update msg ({ date, datePicker } as model) =
                         FailedInput (Disabled d) ->
                             Just <| "Date disabled: " ++ Date.toIsoString d
 
+                        FailedInput (EmptyString) ->
+                            Just <| "Empty string error"
+
```

Comment:
Please remove the unnecessary syntax here: both the brackets and the pipe `<|` is unnecessary.